test.py

```diff
@@ -291,6 +291,8 @@ async def run(self, options):
     def print_summary(self):
         pass
 
+    def get_junit_etree(self):
+        return None
```

Comment:
If I understand this code correctly (and I probably don't...) your code only works on the Boost tests. What about all the other tests which generate xunit result files, e.g., Alternator? Are those files just left alone and not "consolidated" with anything, and at the end their content is visible too?

Reply:
Right.